### DISCUSSION TITLE
Changelog for Images

### DISCUSSION THREAD

**Discussion by michaelklose** [netbox-community/netbox#6159] [Q&A] [answered]
Hello Netbox Community,

in the past we have not used images in NetBox.
Today I have added images to a site and noticed that there is no changelog for it.
Is this working as expected, a bug or would it be a feature request?
I just want to clarify it before opening an issue.

Best Regards
Michael

**Comment by ryanmerolle** [comment, marked as answer]
This is not currently a feature.
